blueocean-material-icons/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@jenkins-cd/blueocean-material-icons",
+  "version": "0.0.5",
+  "description": "Material icons for the Jenkins Design Language",
+  "main": "dist/js/components/index.js",
+  "scripts": {
+    "babel": "node_modules/.bin/babel src -d dist"
```

Comment:
mentioned before, but should be `prepublish` : `npm test && babel src -d dist` or something similar